specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkProfile.json

```diff
@@ -0,0 +1,486 @@
+          "200": {
+            "description": "Request successful. The operation returns a list of NetworkProfile resources.",
+            "schema": {
+              "$ref": "#/definitions/NetworkProfileListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List resource group network profilees": {
+            "$ref": "./examples/NetworkProfileList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkProfilePropertiesFormat": {
+      "properties": {
+        "containerNetworkInterfaces": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerNetworkInterface"
+          },
+          "description": "List of child container network interfaces."
+        },
+        "containerNetworkInterfaceConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerNetworkInterfaceConfiguration"
+          },
+          "description": "List of chid container network interface configurations."
+        }
+      },
+      "description": "Network profile properties."
+    },
+    "NetworkProfile": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkProfilePropertiesFormat",
+          "description": "Network profile properties."
+        },
+        "etag": {
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "Network profile resource."
+    },
+    "NetworkProfileListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkProfile"
+          },
+          "description": "A list of network profiles that exist in a resource group."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Response for ListNetworkProfiles API service call."
+    },
+    "ContainerNetworkInterfacePropertiesFormat": {
+      "properties": {
+        "containerNetworkInterfaceConfiguration": {
+          "$ref": "#/definitions/ContainerNetworkInterfaceConfiguration",
+          "description": "Container network interface configuration from which this container network interface is created."
+        },
+        "container": {
+          "$ref": "#/definitions/Container",
+          "description": "Reference to the conatinaer to which this container network interface is attached."
+        }
+      }
+    },
+    "ContainerNetworkInterface": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ContainerNetworkInterfacePropertiesFormat",
```

Comment:
You probably want to use x-ms-client-flatten to make better SDK.
Example:
https://github.com/Azure/azure-rest-api-specs/blob/32b0d873aa851d456dfde7d6ba1d89ff33f897f0/specification/network/resource-manager/Microsoft.Network/stable/2016-12-01/applicationGateway.json#L511